docs/api/synthetics/monitors/add-monitor-api.asciidoc

```diff
@@ -0,0 +1,168 @@
+[[add-monitor-api]]
+== Add Monitor API
+++++
+<titleabbrev>Add Monitor</titleabbrev>
+++++
+
+Creates a new monitor with the specified attributes. A monitor can be one of the following types: HTTP, TCP, ICMP, or Browser. The required and default fields may vary based on the monitor type.
+
+=== {api-request-title}
+
+`POST <kibana host>:<port>/api/synthetics/monitors`
+
+`POST <kibana host>:<port>/s/<space_id>/api/synthetics/monitors`
+
+=== {api-prereq-title}
+
+You must have `all` privileges for the *Synthetics* feature in the *{observability}* section of the
+<<kibana-feature-privileges,{kib} feature privileges>>.
+
+[[monitor-request-body]]
+==== Request Body
+
+The request body should contain the attributes of the monitor you want to create. The required and default fields differ depending on the monitor type:
+
+*Common Fields*:
+
+`name`:: (Required, string): The monitor's name.
+
+`type`:: (Required, string): The monitor type (e.g., "http," "tcp," "icmp," or "browser").
+
+`schedule`:: (Required, object): The monitor's schedule.
```

Comment:
The way our saved objects are stored is optimized for use with our front end components, but I'm not sure I'd want to utilize the same schema for public api fields. I feel like we should simplify it. For example, schedule could take a simple number of minutes (out of the available options we support, for example 5, 10, 60, 240, etc.

I think this is particularly useful because I believe the "unit" value in schedule is more tech debt that anything, since internally we don't use a "unit" other than minutes. There used to be a seconds unit, but we discarded that after moving towards our 1 minute minimum within the service.